### ISSUE TITLE
Pokemon White (AP Patched+Randomized) freezes during battles.

### ISSUE THREAD

**Issue by MonsterParties** [DS-Homebrew/nds-bootstrap#1638]
### Game Console(s) used

- [ ] Nintendo DS
- [ ] Nintendo DS Lite
- [ ] Nintendo DSi
- [ ] Nintendo DSi XL/LL
- [ ] Nintendo 3DS
- [ ] Nintendo 3DS XL/LL
- [ ] New Nintendo 3DS XL/LL
- [X] Nintendo 2DS
- [ ] New Nintendo 2DS XL/LL

### Launch Method

microSD card with adapter, 32gb. 

### Version used

Release v1.0.0 of nds-bootstrap.
(v26 1.0 of TWiLight Menu++)

### Game tested/used/applicable

Pokemon White, NDS rom dump originally from the cart. AP patched by the RetroGameFan NDS rom patch tool. 
The DSi Binaries are missing post-randomize. 
Randomization by Universal-Randomizer-ZX. 

### Expected Behavior

Enter a battle (wild or trainer), 'fight' option appears on lower screen along with other buttons, touch screen/bottom screen functional.

### Actual Behavior

Every couple of battles, upon the bottom screen menu appearing, the 'fight' button does not appear, and the entire bottom screen is non functional. None of the other options can be selected (bag, run, pokemon). Top screen works fine, battle camera moves around and such, but bottom screen is completely useless. No button inputs work to control it. Forced to reset.

### Steps to reproduce

Encounter pokemon battles through tall grass. 
Defeat pokemon/catch pokemon/flee.
Repeat until glitch occurs.
Occurs randomly, but usually before/around 10 encounters, glitch occurs.
(Also happens in trainer battles randomly, harder to test those repeatedly)

### nds-bootstrap.ini

Do not have an application to open the file, am using a Mac :( 

### Logs

Do not have an application to open the file, am using a Mac :( 

### Will you be available for future issues and/or replies?

Yes

**Comment by RocketRobz** [comment]
Does this still happen on later versions?

**Comment by MonsterParties** [comment]
> Does this still happen on later versions?

Yep. I'm wondering if it's the randomizer though, because it doesn't happen on the default game. The randomizer seems to remove the DSi binaries, and maybe that's what's leading to the glitch.

**Comment by RocketRobz** [comment]
> The randomizer seems to remove the DSi binaries, and maybe that's what's leading to the glitch.

No, that's not why that happens. The game would not boot at all in DSi mode if they we're removed.

**Comment by MonsterParties** [comment]
> No, that's not why that happens. The game would not boot at all in DSi mode if they we're removed.

Ah okay. There's a prompt to boot it in DS mode when I play it, though, which I do.

**Comment by Procamade** [comment]
Hi, I actually have the same issue (on a normal ROM tho, on O3DS and latest version of the bootstrap, which is 1.5.3 as of now), but in my case the freeze on the lower screen always happens on the first battle. I've tried multiple things, like even flashing the DS mode following the guide on the FAQ, but the problem still persists. I know the ROMs aren't the problem since the games work fine on my DSi (btw this happens on all gen V games, gen 4 and others work flawlessly), so I was thinking if downgrading the bootsrap version could help.

Thanks in advance.

**Comment by RocketRobz** [comment]
> Hi, I actually have the same issue (on a normal ROM tho, on O3DS and latest version of the bootstrap, which is 1.5.3 as of now), but in my case the freeze on the lower screen always happens on the first battle. I've tried multiple things, like even flashing the DS mode following the guide on the FAQ, but the problem still persists. I know the ROMs aren't the problem since the games work fine on my DSi (btw this happens on all gen V games, gen 4 and others work flawlessly), so I was thinking if downgrading the bootsrap version could help.
> 
> Thanks in advance.

Is it fixed in later versions?

**Comment by Procamade** [comment]
> > Hi, I actually have the same issue (on a normal ROM tho, on O3DS and latest version of the bootstrap, which is 1.5.3 as of now), but in my case the freeze on the lower screen always happens on the first battle. I've tried multiple things, like even flashing the DS mode following the guide on the FAQ, but the problem still persists. I know the ROMs aren't the problem since the games work fine on my DSi (btw this happens on all gen V games, gen 4 and others work flawlessly), so I was thinking if downgrading the bootsrap version could help.
> > Thanks in advance.
> 
> Is it fixed in later versions?

Can confidently say, that it isn't, atleast for me
